src/views/Statistics.vue

```diff
@@ -83,12 +89,38 @@ export default {
         },
         { name: 'Areas climbed', value: Object.keys(areas).length },
       ];
+    },
+
+    updateItems() {
+      if (!this.userDoc || !this.indexedData || !this.indexedData.routes) {
```

Comment:
i think it'd be better to make sure that we don't go through this function until the team doc (if applicable) has been loaded. otherwise, there will probably be jank on page load where we just generate the non-team stats initially and then generate team stats once the team doc is loaded.

per the above suggestion to return a promise, you can probably `then()` it to set a bool property that's used to bail out until we're done loading.